src/PanoImageRenderer/PanoImageRenderer.js

```diff
@@ -432,6 +445,16 @@ export default class PanoImageRenderer extends Component {
 		this.trigger(EVENTS.BIND_TEXTURE);
 	}
 
+	_updateTexture() {
+		this._renderer.updateTexture(
+			this.context,
+			this.texture,
+			this._image,
+			this._imageConfig,
+		);
+		this._shouldForceDraw = true;
```

Comment:
`shouldForceDraw=true` is not valid. because when user pause  the video, it should not draw a frame.

Reply:
Ok, I'll remove it.